### PR TITLE
PR #172 - Onetime code reformatting.

### PR DESCRIPTION
**Pull request by VboxNick**
The code base has been maintained by various people with different
code style settings which made code formatting inconsistent.
As consequence it's harder to review and follow singly code style.
In PR #173 it has been agreed to do onetime code formatting and then
enforce code style by checkstyle plugin. The checkstyle plugin has
minimal settings and can adjusted anytime by creating a new PR with
proposal.

### CODE REVIEW

File: Server/pom.xml [hunk 1]
```diff
@@ -42,6 +42,15 @@
...
 			</plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemProperties>
+                        <org.apache.commons.logging.Log>org.openas2.logging.Log</org.apache.commons.logging.Log>
+                    </systemProperties>
+                </configuration>
+            </plugin>
```
Comment (igwtech) [inline Server/pom.xml]: The XML has inconsistent identatoin

File: Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java [hunk 2]
```diff
@@ -1,45 +1,47 @@
...
+import org.openas2.cmd.CommandResult;
+
+/**
+ * adds a new partner entry in partnership store
+ *
+ * @author christopher broderick
+ */
+public class SetLogLevelCommand extends AliasedLoggingCommand {
+    public String getDefaultDescription() {
+        return "Set the log level to one of OFF, TRACE, DEBUG, INFO, WARN, ERROR or FATAL." + "\n\tTo disable the override and revert to the configured startup logging levels use RESET." + "\n\tThe level can be set for a specific class by including the class name." + "\n\tNOTE: This command does NOT persist the log level change through a restart of the application.";
```
Comment (igwtech) [inline Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java]: Shouldn't we keep line lengths to a reasonable maximum?

File: Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java [hunk 3]
```diff
@@ -88,30 +86,24 @@ public void init(Session session, Map<String, String> parameters) throws OpenAS2
...
             // create a resource config that scans for JAX-RS resources and providers
             final ResourceConfig rc = new ResourceConfig();
-            rc.register(new LoggerRequestFilter(logger))
-                .register(new AuthenticationRequestFilter(
-                    parameters.getOrDefault("userid","userid"),
-                    parameters.getOrDefault("password","pWd")
-                ))
-                .register(new ApiResource(this))
-                .register(new CORSFilter());
+            rc.register(new LoggerRequestFilter(logger)).register(new AuthenticationRequestFilter(parameters.getOrDefault("userid", "userid"), parameters.getOrDefault("password", "pWd"))).register(new ApiResource(this)).register(new CORSFilter());
```
Comment (igwtech) [inline Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java]: I think splitting fluent API calls into multiple lines makes it more readable

File: Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java (hunk 2 above)
Comment (uhurusurfa) [reply Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java]: Yes - 110 max should be acceptable these days

File: Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java (hunk 3 above)
Comment (uhurusurfa) [reply Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java]: This is ok but for the automated reformat this is probably a manual effort so leave it out if it is and someone can manually change later when working on the file.

File: Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java [hunk 4]
```diff
@@ -1,30 +1,28 @@
...
-	}
-
-	@Override
-	public boolean healthcheck(List<String> failures)
-	{
-		// TODO Auto-generated method stub
-		return true;
-	}
-
-}
```
Comment (igwtech) [inline Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java]: I didn't see the purpose of this file. Seems like left over from an unfinished change. I recommend we remove it from the master branch.

File: Server/pom.xml (hunk 1 above)
Comment (VboxNick) [reply Server/pom.xml]: Sorry, I didn't get you. What is exactly inconsistent?

File: Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java (hunk 3 above)
Comment (VboxNick) [reply Server/src/main/java/org/openas2/cmd/processor/RestCommandProcessor.java]: Intellij can handle this. I will reformat.

File: Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java (hunk 4 above)
Comment (VboxNick) [reply Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java]: I will leave it as it is.Within this PR do formatting and remove comments if needed. No changes. Sorry.

Comment (uhurusurfa) [reply Server/src/main/java/org/openas2/processor/receiver/api/AS2FileReceiverModule.java]: Agreed. This PR is purely for reformatting.

File: Server/pom.xml (hunk 1 above)
Comment (uhurusurfa) [reply Server/pom.xml]: Looks fine to me.

**Comment by uhurusurfa** [comment]
@VboxNick - confirm when you are ready for merge. Looks fine to me at this stage.

**Comment by VboxNick** [comment]
@uhurusurfa sure. I haven't completed with the max length yet. It needs some love.

**Comment by VboxNick** [comment]
[INFO] --- maven-checkstyle-plugin:3.1.0:check (verify-style) @ openas2-server ---
[INFO] There are 520 errors reported by Checkstyle 8.19 with /home/nick/Rnd/Repos/OpenAs2App/Server/target/checkstyle-rules.xml ruleset.
[ERROR] src/main/java/CheckCertificate.java:[59] (sizes) LineLength: Line is longer than 110 characters (found 640).
[ERROR] src/main/java/CheckCertificate.java:[64] (sizes) LineLength: Line is longer than 110 characters (found 123).

520 errors to fix. I will will it as it.

File: Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java (hunk 2 above)
Comment (VboxNick) [reply Server/src/main/java/org/openas2/app/logging/SetLogLevelCommand.java]: there are too much lines > 500 with length more 110 chars. So I didn't change it.

**Comment by VboxNick** [comment]
@uhurusurfa I've completed re-formatting and fixed build.

**Comment by igwtech** [comment]
Great work Nick!
 Thanks. :)

**Comment by uhurusurfa** [comment]
@VboxNick - thanks Nick

**Pull request merged**